modules/administration-guide/pages/finding-the-fully-qualified-domain-name-fqdn.adoc

```diff
@@ -0,0 +1,28 @@
+:_content-type: PROCEDURE
+:description: Finding the fully qualified domain name (FQDN)
+:keywords: administration-guide, FQDN, URL
+:navtitle: Finding the fully qualified domain name (FQDN)
+//:page-aliases:
+
+[id="finding-the-fully-qualified-domain-name-fqdn"]
+= Finding the fully qualified domain name (FQDN)
+
+You can get the fully qualified domain name (FQDN) of your organization's instance of {prod-short} on the command line or in the OpenShift web console.
+
+TIP: You can find the FQDN for your organization's {prod-short} instance in the *Administrator* view of the OpenShift web console as follows. Go to *Operators* -> *OperatorHub* -> *Installed Operators* -> *{prod} instance Specification* -> *{prod-checluster}* -> *{prod} URL*.
+
+
+.Prerequisites
+
+* An active `oc` session with administrative permissions to the OpenShift cluster. See link:https://docs.openshift.com/container-platform/{ocp4-ver}/cli_reference/openshift_cli/getting-started-cli.html[Getting started with the OpenShift CLI].
+* `{prod-cli}`. See xref:installing-the-chectl-management-tool.adoc[].
+
+.Procedure
+
+. Run the following command:
++
+[subs="+attributes,+quotes"]
+----
+$ {prod-cli} server:status
+----
+. Copy the *{prod} URL* without the trailing `/dashboard/`.
```

Comment:
Is this term *{prod} URL* an element in the terminal? Do we need to bold this element?

Reply:
Yes, and it is already bold: `*{prod} URL*`.